### PR TITLE
Make sure that we fail when given bad input

### PR DESCRIPTION
**Pull request by sunesimonsen**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -20,12 +20,20 @@ export function unmount(element) {
 }
 
 export function simulate(rootElement, events) {
+  if (arguments.length !== 2) {
+    throw new Error("simulate take exactly two arguments");
```
Comment (zenpinto) [inline src/index.js]: I would make "take" plural. "simulate takes exactly two arguments"

File: src/index.js [hunk 2]
```diff
@@ -20,12 +20,20 @@ export function unmount(element) {
...
+  }
+
   []
     .concat(events)
     .map(event => (typeof event === "string" ? { type: event } : event))
     .forEach(event => {
       let target = rootElement;
 
+      if (!event.type) {
+        throw new Error("All event must have a type");
```
Comment (zenpinto) [inline src/index.js]: Same here. "All events must have a type"

**Comment by sunesimonsen** [comment]
I'll just merge this as it is uncontroversial.

**Comment by sunesimonsen** [comment]
Released as react-dom-testing@1.8.0

**Pull request merged**